src/views/Map/index.tsx

```diff
@@ -46,6 +48,14 @@ export const MapView = ({ }: RouteComponentProps) => {
     if (replaceDataModalOpen) setReplaceDataModalOpen(false)
   }
 
+  const refreshIframe = () => {
+    setIframeLoading(true)
+    setTimeout(() => {
+      // setIframeKey((key) => key + 1)
+      setIframeLoading(false)
+    }, 1000)
+  }
+
```

Comment:
this currently is not actually refreshing the Iframe, is it?

Reply:
It should be...

Reply:
maybe I dont understand the architecture fully, but currently you are just mounting and unmounting the i frame. Is that all your trying to accomplish here?

Reply:
yes

Reply:
but we need to do it with a delay for the data to update, I don't remember the details of why anymore but it wasn't updating the data in the iframe unless we delay it. The workaround was to show a loader for 1 second

Reply:
no, I think the current code is fine. I just wanted clarification because I wasn't sure if that was intentional as The `setIframeKey((key) => key + 1)` was confusing me because I thought that was resetting the the i frame. 

Also lets erase that commented out code.